kqueen_ui/asset/dynamic/scss/main.scss

```diff
@@ -647,6 +657,58 @@ td.mdi-account {
   margin-right: 3px;
 }
 
+div.bulk-action {
+  text-align: right;
+  margin-top: -5px;
+}
+
+.table > tbody > tr > td.clickable-cluster-name {
```

Comment:
doesn't connected with bulk commit, I suggest to save color to our variable list

Reply:
It is connected to this commit because I've changed the behaviour of row click to the checkbox selection, but for names, instead, I made the whole cell clickable. So this cell should be highlighted.
As for the color, not sure. It's used only here and probably will not be used anywhere else